api/jobimp/v1/jobimp.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package api.jobimp.v1;
+
+option go_package = "api/jobimp/v1";
+
+service JobimpService {
+  rpc ImportJob(JobimpRequest) returns (JobimpResponse) {}
+}
+
+message Location {
+  double lat = 1;
+  double lon = 2;
+}
+
+message JobimpRequest {
+  string id = 1;
+  string worker_id = 2;
+  google.protobuf.Timestamp created = 3;
+  google.protobuf.Timestamp updated = 4;
+  string state = 5;
+  google.protobuf.Timestamp pickup_date = 6;
+  string pickup_addr = 7;
+  Location pickup_loc = 8;
+  string dropoff_addr = 9;
+  Location dropoff_loc = 10;
+  string trip_type = 11;
+  string category = 12;
+}
+
+enum JobimpAction {
+  JOBIMP_UNKNOWN = 0;
```

Comment:
I **think** the linter will want this to be `JOBIMP_ACTION_UNSPECIFIED`

https://docs.buf.build/lint/rules#enum_zero_value_suffix

Reply:
Fixed. Thanks.